zerver/lib/validator.py

```diff
@@ -632,6 +634,25 @@ class WildValue:
     var_name: str
     value: object
 
+    @model_validator(mode="wrap")  # type: ignore[arg-type] # The upstream's type annotation uses a TypeVar that is incorrectly unbounded.
+    @classmethod
+    def to_wild_value(
+        cls,
+        value: Any,
+        # We bypass the original WildValue handler to customize it
+        handler: ModelWrapValidatorHandler["WildValue"],
+        info: ValidationInfo,
+    ) -> "WildValue":
+        # TODO: Move WildValue elsewhere to fix circular import
+        from zerver.lib.typed_endpoint import get_func_param
+
+        var_name = get_func_param(info).request_var_name
```

Comment:
The `get_func_param` context mechanism adds more complexity than is warranted to compute a parameter name that (a) is only used in error messages that the user will never see, (b) doesn’t even name a real parameter since it’s the request body, and (c) is always `payload`, as far as I can see.

We can just hard-code `payload`.